get_github_coderabbit_review/coderabbit_fetcher/analyzers/structured_comment_parser.py

```
@@ -0,0 +1,527 @@
+        """
+        comments = []
+
+        # Outside Diff Rangeセクションの場合、直接パターンマッチング
+        if section_type == CommentSection.OUTSIDE_DIFF_RANGE:
+            # 直接section_content全体から`行番号`: **タイトル** パターンを検索
+            direct_matches = self.line_comment_pattern.finditer(section_content)
+            for match in direct_matches:
+                line_range = match.group(1)
+                title = match.group(2)
+
+                # ファイル名を抽出（summary内から）
```

Ruff (0.13.1)

Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)